development/kops/set_environment.sh

```diff
@@ -61,7 +61,7 @@ fi
 export PREFLIGHT_CHECK_PASSED
 
 export KUBERNETES_VERSION=$(cat ../../projects/kubernetes/kubernetes/${RELEASE_BRANCH}/GIT_TAG)
-export DEFAULT_REPOSITORY_URI=public.ecr.aws/eks-distro
+export DEFAULT_REPOSITORY_URI=public.ecr.aws/h1r8a7l5
```

Comment:
This default repository is correct. Use REPOSITORY_URI to whatever repo, but no need to change this.

I think the reason we had DEFAULT_REPOSITORY_URI is no nore and we could get rid of it, but no big deal